openpype/hosts/maya/tools/mayalookassigner/commands.py

```diff
@@ -80,21 +80,9 @@ def get_all_asset_nodes():
     Returns:
         list: list of dictionaries
     """
-
-    host = registered_host()
-
-    nodes = []
-    for container in host.ls():
-        # We are not interested in looks but assets!
-        if container["loader"] == "LookLoader":
-            continue
-
-        # Gather all information
-        container_name = container["objectName"]
-        nodes += lib.get_container_members(container_name)
-
-    nodes = list(set(nodes))
-    return nodes
+    nodes = cmds.ls(dag=True, noIntermediate=True, long=True)
+    items = create_items_from_nodes(nodes)
```

Comment:
Isn't the `"LookLoader"` filtering important?

Reply:
The new way doesn't check against containers at all. But in a way somewhat important. This is what I mentioned in the PR description:

> IF an already loaded look has any dagNodes, say a 3D Projection node - then that will also show up as a loaded asset where previously nodes from loaded looks were ignored.
> If any dag nodes were created locally - they would have gotten cbId attributes on scene save and thus the current asset would almost always show?

It filters to dag nodes only which tend to not be present in many looks. It could do a more thorough check to filter out look nodes, however the "from selection" method also doesn't filter out look nodes. (But maybe you're not likely to select them.)

We could do a full container membership lookup to filter out look nodes again if needed - but it'd be quite a bit slower. I heard there was a UI redesign in the works for the Look Assigner (right @mkolar). I think for that design it might be nice to just filter by families of content (so e.g. it'd only show pointcaches if you wanted it to.) Which could then have disabled the `look` family by default. 👍

Reply:
@antirotor could you still validate whether this is currently ok and still behaves in an acceptable manner for Get All Assets?